README.md

```diff
@@ -1,4 +1,50 @@
 # nvmconverter
 Converts the VisualSFM .nvm (N-View Match) filetype to the OpenSFM .json (JavaScript Object Notation) file type. The NVM file type is discribed here: [vsfm nvm](http://ccwu.me/vsfm/doc.html#nvm)
 
+# OpenSfM
+There is a Docker image of OpenSfM available by searching 
+`docker search awkbr549/opensfm'
 
+To run the image, it is advised to run with the following options:
+`-it'		--> used with the `run' command, creates an interactive and pseudo-tty connection to the container
+`-ai'		--> used with the `start' command to allow interaction with an already existing Docker container
+`-p 8000:8000'		--> maps Docker container port 8000 to localhost 8000
+`-v dir/on/local/computer:dir/on/container' 	--> maps local files to the container
+
+On Windows, Docker's option to map local files to the container doesn't work (at least for me). In order to get around this, you can just use the `scp' file transfer command to move files from your local machine to a server, then also in your Docker container use `scp' to transfer files from the server to the container. Though cumbersome, this is the only workaround I've found.
+
+Example command:
+`docker run -it -p 8000:8000 awkbr549/opensfm:v2' //for new containers (!!! should only run once)
```

Comment:
ok, it looks like you just consistently have the error of starting a code block with ` and ending it with '